gadget/params.c

```diff
@@ -9,6 +9,10 @@
 #include <libgadget/utils.h>
 #include <libgadget/treewalk.h>
 #include <libgadget/cooling_rates.h>
+#include <libgadget/winds.h>
+#include <libgadget/sfr_eff.h>
+#include <libgadget/blackhole.h>
+#include <libgadget/fof.h>
```

Comment:
yikes. This is how it should looks like. Real header files!

Reply:
Yup!